specifications/api/swagger.yaml

```diff
@@ -5590,6 +5590,19 @@ x-errUnresolvedInputs: &errUnresolvedInputs
       type: string
       enum: ['unknown_inputs']
 
+x-errUnresolvedRefunds: &errUnresolvedRefunds
+  <<: *responsesErr
+  title: unresolved_refunds
+  properties:
+    message:
+      type: string
+      description: |
+        There are stake key deregistration certificates in the transaction for which corresponding
+        deposit amounts could not be found.
+    code:
+      type: string
+      enum: ['unresolved_refunds']
+
```

Comment:
Had to add this to keep the test happy, but IMO we shouldn't need to concern the API specification about err500 errors